src/Handlers/ApiHandlerInterface.php

```diff
@@ -68,4 +68,11 @@ public function setEndpointIdentifier(EndpointInterface $endpoint): void;
      * @return OutputInterface[]
      */
     public function outputs(): array;
+
+    /**
+     * List of additional information for handler
+     *
+     * @return array<string, int|string|bool|array>
+     */
+    public function additionalData(): array;
```

Comment:
hm... it is possible BC if there are Handlers implemented directly this interface instead of BaseHandler

Reply:
thinking about this change. not sure if this is a way because:

1. it is BC break as you wrote here
2. we are changing the interface, but the use case for this change is in specific implementation in OpenApiHandler.

Reply:
> 2. we are changing the interface, but the use case for this change is in specific implementation in OpenApiHandler.

We are not implementing method additionalData() in OpenApiHandler. It is just used there from all Handlers to generate documentation

Reply:
just for additionalData method? we can, but it is little bit confusing, isn't it?

Reply:
it will make sense when we will put there all related stuff ;-)

Reply:
well:
1. it would be also BC break because user will have to use this new interface in handlers to cover all this stuff
2. mentioned methods (summary, description, etc.) are also used in api console / api listing controls so they are not "open api" specific